src/styles/card.css

```diff
@@ -0,0 +1,48 @@
+.card {
+  width: 198px;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px 0;
+  margin-bottom: 36px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  padding: 0 16px;
+  color: #060b35;
+  margin: 0 0 4px;
+  line-height: 18px;
+}
+
+.card__model {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  padding-left: 16px;
+  margin: 0 0 17px;
+}
+
+.card__buy-button {
+  width: 166px;
+  height: 40px;
+  margin: 0 16px 16px;
+  background: #00acdc;
```

Comment:
you typed the #00acdc 3 times
but what if we decide to use another color?
we would need to find all the places we use it and change
if you use CSS variable you can change the color in 1 place

Reply:
do you mean I should add input for button click ? or is there any attribute I should add to the button element to add a link src ?